cmd/migrate_intakes/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
+	"gopkg.in/launchdarkly/go-server-sdk.v5/ldcomponents"
+	"gopkg.in/launchdarkly/go-server-sdk.v5/testhelpers/ldtestdata"
+
+	"github.com/cmsgov/easi-app/pkg/appconfig"
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/storage"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+)
+
+func noErr(err error) {
+	if err != nil {
+		fmt.Println("Error!")
+		fmt.Println(err)
+		panic("Aborting")
+	}
+}
+
+func makeStore(ldClient *ld.LDClient) *storage.Store {
+	config := testhelpers.NewConfig()
+
+	dbConfig := storage.DBConfig{
+		Host:           config.GetString(appconfig.DBHostConfigKey),
+		Port:           config.GetString(appconfig.DBPortConfigKey),
+		Database:       config.GetString(appconfig.DBNameConfigKey),
+		Username:       config.GetString(appconfig.DBUsernameConfigKey),
+		Password:       config.GetString(appconfig.DBPasswordConfigKey),
+		SSLMode:        config.GetString(appconfig.DBSSLModeConfigKey),
+		MaxConnections: config.GetInt(appconfig.DBMaxConnections),
+	}
+
+	store, storeErr := storage.NewStore(dbConfig, ldClient)
+	if storeErr != nil {
+		panic(storeErr)
+	}
+	return store
+}
+
+func migrateIntakes() {
+	zapLogger, err := zap.NewDevelopment()
+	noErr(err)
+
+	ctx := appcontext.WithLogger(context.Background(), zapLogger)
+
+	td := ldtestdata.DataSource()
+	td.Update(td.Flag("emit-to-cedar").BooleanFlag().VariationForAll(true))
+	config := ld.Config{
+		DataSource: td,
+		Events:     ldcomponents.NoEvents(),
+	}
+
+	ldClient, err := ld.MakeCustomClient("fake", config, 0)
+	if err != nil {
+		fmt.Println(err)
+		panic("Error initializing ldClient")
+	}
+
+	store := makeStore(ldClient)
+
+	// Fetch all intakes
+	fmt.Println("Fetching all system intakes")
+	fmt.Println("=======================================")
+	allIntakes, err := store.FetchSystemIntakes(ctx)
+	noErr(err)
+	for _, i := range allIntakes {
+		intake := i // prevent gosec's G601 -- Implicit memory aliasing in for loop.
```

Comment:
Is this that thing that got handled in 1.21?

Reply:
It's not addressed in 1.21 without enabling experimental features, so this will likely get addressed in 1.22 or on, so I just left a comment of G601 so we can loop back to it when that's all fixed